backend/plugins/AGENT.md

```
@@ -0,0 +1,73 @@
+# Express Middleware
+
+Uusgesen apollo server iig Express app tai holboj huseltuudiig udirdana.
+
+## Mongoose Models
+
+definition hesegt "timestamps: true" gej ugvul "createdAt", "updatedAt" automataar uusdeg.
+id-uudiig zaaval indexleh ystoi.
+
+## Trpc
+
+        prevMessage = await sendTRPCMessage({
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)